scripts/generate-yaml-tasks.sh

```diff
@@ -34,6 +38,11 @@ function generateTaskResources() {
         DESTINATION_PARENT_DIR="${REPO_DIR}/tasks/${TASK_NAME}"
         DESTINATION="${DESTINATION_PARENT_DIR}/${RESOURCE_TYPE}"
         SOURCE="${REPO_DIR}/templates/${TASK_NAME}/dist/${RESOURCE_TYPE}"
+        
```

Comment:
in case you would go this way, I think it is preferable to read an env variable in the ansible and set tag accordingly instead of doing post processing like this.

Also you can either remove the tag from configs or just create a new `main_image_tag` ansible variable there and then either use that or switch to the env variable accordingly.